### ISSUE TITLE
loop on INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...

### ISSUE THREAD

**Issue by johndpope** [lancopku/Skeleton-Based-Generation-Model#1]
(tf14) ➜  Skeleton-Based-Generation-Model git:(master) ✗ ./run_train.sh
/Users/jpope/miniconda2/envs/tf14/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: compiletime version 3.5 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.6
  return f(*args, **kwds)
/Users/jpope/miniconda2/envs/tf14/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
/Users/jpope/miniconda2/envs/tf14/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192, got 176
  return f(*args, **kwds)
INFO:tensorflow:Starting running in train mode...
max_size of vocab was specified as 20000; we now have 20000 words. Stopping reading.
Finished constructing vocabulary of 20000 total words. Last word added: richie
Start pre-training......
INFO:tensorflow:Building generator graph...
INFO:tensorflow:Time to build graph: 2 seconds
2018-08-20 12:36:03.262610: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
INFO:tensorflow:Failed to load checkpoint from log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/log_autoencoder/myexperiment/train-sc-generator. Sleeping for 10 secs...
^Z
[5]  + 50353 suspended  ./run_train.sh

**Comment by johndpope** [comment]
conda / tensorflow environment
https://gist.github.com/johndpope/2322c10951579d632ecf752a7a36b559

```shell
pip install spacy && python -m spacy download en
./run_train.sh
```

**Comment by jingjingxupku** [comment]
The problem is fixed. Please update the source code.

**Comment by johndpope** [comment]
thanks - it's working.
